nvflare/private/fed/client/admin_msg_sender.py

```diff
@@ -169,3 +169,6 @@ def _set_up_channel(self, channel_dict):
         else:
             channel = grpc.insecure_channel(**channel_dict)
         return channel
+
+    def close(self):
+        self.pool.close()
```

Comment:
should you do 
    pool.close()
    pool.join()
https://stackoverflow.com/questions/38271547/when-should-we-call-multiprocessing-pool-join